waku/v2/node/storage/peer/waku_peer_storage.nim

```diff
@@ -0,0 +1,136 @@
+import
+  std/sets, 
+  sqlite3_abi,
+  chronos, metrics,
+  libp2p/protobuf/minprotobuf,
+  stew/results,
+  ../sqlite,
+  ../../peer_manager
+
+export sqlite
+
+type
+  WakuPeerStorage* = ref object of RootObj
```

Comment:
Is this a separate DB from the one used for store? How will it interact with the whole sqlite/encrypted mismatch we saw with Stimbus previously?

Reply:
The idea is to use the same DB and just add a separate table. I foresee the same clash as previously, so will similarly extract an interface for Stimbus to provide their own (optional) PeerStorage.